Portal/Views/Waste/Index.cshtml

```diff
@@ -0,0 +1,44 @@
+﻿@using Portal.Resources;
+@{
+    ViewData["Title"] = HomeResources.HomePage;
+}
+
+<div class="text-center">
+    <a href="#" class="govuk-back-link">Back</a>
+</div>
+
+<div class="govuk-form-group">
+    <fieldset class="govuk-fieldset" aria-describedby="waste-hint waste-error">
+        <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+            <h1 class="govuk-fieldset__heading">
+                What have you done with the waste?
+            </h1>
+        </legend>
+        <div id="waste-hint" class="govuk-hint">
+            Hint text.
+        </div>
+        <p id="waste-error" class="govuk-error-message">
+            <span class="govuk-visually-hidden">Error:</span>Please select a radio button
+        </p>
+        <div class="govuk-radios" data-module="govuk-radios">
+            <div class="govuk-radios__item">
+                <input class="govuk-radios__input" id="whatHaveYouDoneWithWaste-0" name="whatHaveYouDoneWithWaste" type="radio" value="repoceesedIt">
+                <label class="govuk-label govuk-radios__label" for="whereDoYouLive">
+                    Received it and reproceesed it
+                </label>
+            </div>
+            <div class="govuk-radios__item">
+                <input class="govuk-radios__input" id="whatHaveYouDoneWithWaste-1" name="whatHaveYouDoneWithWaste" type="radio" value="sentItOn">
+                <label class="govuk-label govuk-radios__label" for="whereDoYouLive-2">
+                    Received it and sent it on
+                </label>
+            </div>
+        </div>
+        <p></p>
+        <div class="govuk-button-group">
+            <button class="govuk-button" data-module="govuk-button">
+                Continue
+            </button>
+        </div>
+    </fieldset>
+</div>
```

Comment:
The text used here should come from Resource files rather than written directly in the view. when supplying the welsh just use the English and prefix it with [Welsh] so that we can do a find on everything untranslated.

Reply:
Done.